### PR TITLE
📝Update Readme.md (Code Error Minimal Example)

### PR DESCRIPTION
**Pull request by pawarashish564**
to use gpus we need to use
```
trainer = pl.Trainer(max_epochs=1, gpus=8)
```
but In readme only Trainer() is used

## What does this PR do?

I fixed the code in Readme.

I tried to run the code given in the readme or minimal example. To use GPu we need to use pl.Trainer() not Trainer()
# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure your PR does only one thing, instead of bundling different changes together? Otherwise, we ask you to create a separate PR for every change.
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? 
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## PR review    
Please anyone review my change and let me know what you think about this.

### CODE REVIEW

**Comment by williamFalcon** [comment]
I'd prefer to not add the pl.Trainer... adds confusion to the readme

**Comment by awaelchli** [comment]
we could add `from pytorch_lightning import Trainer` to the import statements above, maybe?

**Comment by pawarashish564** [comment]
Yes sure we can do this also.
```from pytorch_lightning import Trainer
```
I know pl.Trainer() is kinda complex but For new devs who didn't have any idea about the Pytorch Lightning. 
We should provide a proper snippet that they can run without any errors.

**Comment by williamFalcon** [comment]
yes, i'd prefer to just add the import

**Review by Borda — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -201,6 +201,8 @@ trainer.fit(model, DataLoader(train), DataLoader(val))
 #### And without changing a single line of code, you could run on GPUs
 ```python
 # 8 GPUs
+from pytorch_lightning import Trainer
```
Comment (awaelchli) [inline README.md:204]: can we simply add it to the code snippet above, because these lines here refer to the example above.

Comment (pawarashish564) [reply README.md:204]: ok I add it all import statements above.

**Comment by pawarashish564** [comment]
@awaelchli See now .
less code change the added the import to all imports.

**Review by SkafteNicki — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -194,13 +195,15 @@ dataset = MNIST(os.getcwd(), download=True, transform=transforms.ToTensor())
...
-trainer = pl.Trainer()
+trainer = Trainer()
 trainer.fit(model, DataLoader(train), DataLoader(val))
 ```
 
 #### And without changing a single line of code, you could run on GPUs
 ```python
 # 8 GPUs
+
+
```
Comment (awaelchli) [inline README.md:206]:
```suggestion
```
not needed, right?

**Review by awaelchli — APPROVED** [review]
:+1:

**Pull request merged**